helm/cas-ciip-portal/values.yaml

```diff
@@ -59,8 +59,8 @@ dag:
 
 env:
   smtpConnectionString: smtp://apps.smtp.gov.bc.ca/?port=25&ignoreTLS=true&secure=false
-  senderEmail: BCCAS <no-reply.cas@gov.bc.ca>
-  adminEmail: GHG Regulator <GHGRegulator@gov.bc.ca>
+  senderEmail: no-reply.cas@gov.bc.ca
+  adminEmail: GHGRegulator@gov.bc.ca
```

Comment:
Never noticed it before, but I wonder why we had the much more nicely spelled-out sender/receiver names in dev and test, but not in prod? 
ie: "CIIP Admin, BC Climate Action Secretariat" and "BC Climate Action Secretariat" in values-test and values-dev

Maybe we should be update [the lines in `sendMail`](https://github.com/bcgov/cas-ciip-portal/pull/1610/files#diff-a4b1bb3c67981694cbd9ddc2073beb8a68075b86310c6054cb26e778f4aad76dR15-R16) that hardcode the name part to this instead...

Reply:
Yeah, maybe taking the information from values-test as that is what people saw when doing QA.